src/DFG.cpp

```diff
@@ -30,9 +31,9 @@ DFG::DFG(Function& t_F, list<Loop*>* t_loops, bool t_targetFunction,
   construct(t_F);
   if (t_heterogeneity) {
```

Comment:
I remember we mentioned using another flag to skip Jiajun's work? Or now, both works can co-exist, and both controlled by this `t_heterogeneity`?

Reply:
What about using string list?
 - Rename it to `fusion_strategy`
   - `fusion_strategy` not provided or provided as `fusion_strategy = []`, indicating `do nothing`
   - `fusion_strategy = ["default_heterogeneous"]`
   - `fusion_strategy = ["nonlinear"]`
   - `fusion_strategy = ["ctrl_flow"]`
   - `fusion_strategy = ["unroll"]`
   - `fusion_strategy = ["nonlinear", "ctrl_flow"]`